packages/fx-core/resource/yaml-schema.json

```diff
@@ -0,0 +1,1495 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "projectId": {
+      "type": "string",
+      "description": "projectId used for telemetry"
+    },
+    "environmentFolderPath": {
+      "type": "string",
+      "description": "path to the environment folder"
+    },
+    "version": {
+      "type": "string",
+      "description": "The version of the yaml file schema"
+    },
+    "provision": {
+      "$ref": "#/definitions/lifeCycleArray",
+      "description": "Triggered by `teamsfx provision`"
```

Comment:
I think the word "Called" is better then "Triggered"?

Reply:
Done